### PR TITLE
player_thrusts

### PR DESCRIPTION
**Pull request by Miho42**
Player has functioning rocket engine 

> - 0 -

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -254,9 +254,16 @@ def on_update(self, delta_time):
         elif self.right_pressed and not self.left_pressed:
             self.player_sprite.angle -= PLAYER_ROTATE_SPEED
 
+        # Player rocket engine
+        if self.up_pressed:
+            # rotate player graphics to match direction
+            self.player_sprite.change_x += self.player_sprite.speed * cos(self.player_sprite.radians + pi/2)
+            self.player_sprite.change_y += self.player_sprite.speed * sin(self.player_sprite.radians + pi/2)
+
```
Comment (tobiasbp) [inline my_game.py:262]: We should probably have a global variable _PLAYER_THRUST_ which should replace _self.speed_ used here.

File: my_game.py [hunk 2]
```diff
@@ -254,9 +254,16 @@ def on_update(self, delta_time):
...
+        if self.up_pressed:
+            # rotate player graphics to match direction
+            self.player_sprite.change_x += self.player_sprite.speed * cos(self.player_sprite.radians + pi/2)
+            self.player_sprite.change_y += self.player_sprite.speed * sin(self.player_sprite.radians + pi/2)
+        
+
         # Move player with joystick if present
         if self.joystick:
-            self.player_sprite.change_x = round(self.joystick.x) * PLAYER_SPEED_X
+            self.player_sprite.change_x = round(self.joystick.x) * PLAYER_SPEED
```
Comment (tobiasbp) [inline my_game.py:266]: This code is wrong. Test with a joypad, and make sure it also works for that.

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py (hunk 2 above)
Comment (Miho42) [reply my_game.py:266]: Don't want to :(

I will make a new issue, and get another person to fix my problems :+1:

File: my_game.py (hunk 1 above)
Comment (Miho42) [reply my_game.py:263]: is fixed?

Comment (Miho42) [reply my_game.py:252]: Is berry fixed

**Review by tobiasbp — APPROVED** [review]

**Pull request merged**